src/qibolab/platform.py

```diff
@@ -132,18 +171,14 @@ def disconnect(self):
                 instrument.disconnect()
         self.is_connected = False
 
+    @property
+    def controller(self):
```

Comment:
This is missing a docstring.

Please describe even the assertion error in the docstring.